src/api/auth.ts

```diff
@@ -118,6 +118,12 @@ export function getTeamUsers(team: string, organization: string, auth: any): any
   return users;
 }
 
+export function checkPackageName(pkgName: string): boolean {
+  let regexStart = /^[a-z0-9~()'!*:@,;+?-]{1}/g;
```

Comment:
yes, I think `~:?` shouldn't be allowed. can you maybe make single regex for this checking?